.gitignore

```diff
@@ -1,2 +1,7 @@
 urdfEnvs.egg-info/
 *.swp
+
+# ignore intellij config files
+.idea
+idea/
```

Comment:
Do we need those ignore items?
It might be related to your local settings.

Reply:
We do not need all the files in the .idea folder which are: 
(venv) gijs@gijs-HP-ZBook-Studio-x360-G5:~/Documents/gym_envs_urdf/.idea$ ls
gym_envs_urdf.iml  inspectionProfiles  misc.xml  modules.xml  other.xml  vcs.xml  workspace.xml

That is why I ignored everything in the .gitignore

I will put this in the .gitignore inside the .idea/ folder. Then I will not appear in the repository.